package.json

```diff
@@ -188,14 +193,13 @@
 		"fixtures:generate": "npm run fixtures:server-registered && cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit",
 		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
 		"format-js": "wp-scripts format-js",
-		"lint": "concurrently \"npm run lint-js\" \"npm run lint-pkg-json\" \"npm run lint-css\" \"npm run lint-types\"",
+		"lint": "concurrently \"npm run lint-js\" \"npm run lint-pkg-json\" \"npm run lint-css\"",
 		"lint-js": "wp-scripts lint-js",
 		"lint-js:fix": "npm run lint-js -- --fix",
 		"lint-php": "npm run wp-env run composer run-script lint",
 		"lint-pkg-json": "wp-scripts lint-pkg-json . 'packages/*/package.json'",
 		"lint-css": "wp-scripts lint-style '**/*.scss'",
 		"lint-css:fix": "npm run lint-css -- --fix",
-		"lint-types": "tsc",
```

Comment:
From the original comment:

>Remove `lint-types` script, replaced by `build:packages-types`.

Just to confirm, will `build:packages-types` adequately fail builds where we're expecting type-checking to occur?

Reply:
Yes, `npm run build:package-types` will error. It's part of `build:packages`. Should we add that to `lint`? That runs on the precommit hook?

Reply:
Yes, I think it would be good to have both as a pre-commit task, and reflected in the Travis build as well.